app/views/shared/check_answers/_vehicles.html.erb

```diff
@@ -9,38 +9,46 @@
-        answer: gds_number_to_currency(@legal_aid_application.vehicle.estimated_value),
-        read_only: read_only,
-      ) if @legal_aid_application.own_vehicle? %>
+  <%= if @legal_aid_application.own_vehicle?
+        check_answer_link(
+          name: :vehicles_estimated_values,
+          url: check_answer_url_for(journey_type, :vehicles_estimated_values, @legal_aid_application),
+          question: t(".#{journey_type}.estimated_value"),
+          answer: gds_number_to_currency(@legal_aid_application.vehicle.estimated_value),
+          read_only:,
+        )
+      end %>
 
-  <%= check_answer_link(
-        name: :vehicles_remaining_payments,
-        url: check_answer_url_for(journey_type, :vehicles_remaining_payments, @legal_aid_application),
-        question: t(".#{journey_type}.payment_remaining"),
-        answer: gds_number_to_currency(@legal_aid_application.vehicle.payment_remaining),
-        read_only: read_only,
-      ) if @legal_aid_application.own_vehicle? %>
+  <%= if @legal_aid_application.own_vehicle?
+        check_answer_link(
+          name: :vehicles_remaining_payments,
+          url: check_answer_url_for(journey_type, :vehicles_remaining_payments, @legal_aid_application),
+          question: t(".#{journey_type}.payment_remaining"),
+          answer: gds_number_to_currency(@legal_aid_application.vehicle.payment_remaining),
+          read_only:,
+        )
+      end %>
 
-  <%= check_answer_link(
-        name: :vehicles_ages,
-        url: check_answer_url_for(journey_type, :vehicles_ages, @legal_aid_application),
-        question: t(".#{journey_type}.more_than_three_years_old"),
-        answer: yes_no(@legal_aid_application.vehicle.more_than_three_years_old?),
-        read_only: read_only,
-      ) if @legal_aid_application.own_vehicle? %>
+  <%= if @legal_aid_application.own_vehicle?
+        check_answer_link(
+          name: :vehicles_ages,
+          url: check_answer_url_for(journey_type, :vehicles_ages, @legal_aid_application),
+          question: t(".#{journey_type}.more_than_three_years_old"),
+          answer: yes_no(@legal_aid_application.vehicle.more_than_three_years_old?),
+          read_only:,
+        )
+      end %>
 
-  <%= check_answer_link(
-        name: :vehicles_regular_uses,
-        url: check_answer_url_for(journey_type, :vehicles_regular_uses, @legal_aid_application),
-        question: t(".#{journey_type}.used_regularly"),
-        answer: yes_no(@legal_aid_application.vehicle.used_regularly),
-        read_only: read_only,
-      ) if @legal_aid_application.own_vehicle? %>
+  <%= if @legal_aid_application.own_vehicle?
+        check_answer_link(
+          name: :vehicles_regular_uses,
+          url: check_answer_url_for(journey_type, :vehicles_regular_uses, @legal_aid_application),
+          question: t(".#{journey_type}.used_regularly"),
+          answer: yes_no(@legal_aid_application.vehicle.used_regularly),
+          read_only:,
+        )
+      end %>
```

Comment:
given that these are all testing the one condition and are already sequential we would it work to have one condition with a all of the content yielded or somesuch.